flake-lang/build.nix

```diff
@@ -81,22 +104,6 @@
             example = lib.mdDoc ''TODO(jaredponn)'';
           };
 
-          rustMonorepoPreCommit = lib.mkOption {
-            type = lib.types.flakeModule;
-            default = ./pre-commit-hooks/rust-monorepo.nix;
-            readOnly = true;
-            description = lib.mdDoc ''pre-commit-hooks.nix hook for Rust in a monorepo setting'';
-            example = lib.mdDoc ''TODO(bladyjoker)'';
-          };
-
-          denoPreCommit = lib.mkOption {
-            type = lib.types.flakeModule;
-            default = ./pre-commit-hooks/deno.nix;
-            readOnly = true;
-            description = lib.mdDoc ''pre-commit-hooks.nix hook for Deno in a monorepo setting'';
-            example = lib.mdDoc ''TODO(bladyjoker)'';
-          };
-
```

Comment:
We don't need to roll our own deno anymore. They fixed this in the main repo

Reply:
Although this prompts some discussion. In Rust, it seems we want to export a module which does all the formatting -- do we want this for all languages (or none)?

Reply:
Not sure what you mean...

AFAIK this is about providing Rust formatting with pre-commit-checks.nix, there were reasons why we had to roll our own. @szg251 any comments?